.github/workflows/tests.yml

```diff
@@ -465,6 +465,8 @@ jobs:
           POSTGRES=${{ (matrix.database == 'Postgres') && 1 || '' }} WORKERS=${{ (matrix.arrangement == 'workers') && 1 || '' }} COMPLEMENT_DIR=`pwd`/complement synapse/scripts-dev/complement.sh -json 2>&1 | synapse/.ci/scripts/gotestfmt
         shell: bash
         name: Run Complement Tests
+        env:
+          COMPLEMENT_ALWAYS_PRINT_SERVER_LOGS: 1
```

Comment:
It is.
I mention reverting it once everything is reconciled.
>There are two commits in this pull request that can be reverted. The one is harmless and probably would be good to keep. The other is bad and shouldn't be used long term on Github(tons of log spam):

>* A line is output when Generating base homeserver config that didn't have a mate when config already exists. I added that in before collecting logs while diagnosing the issue. Look for Homeserver config already exists at:.
>* Added the environment variable COMPLEMENT_ALWAYS_PRINT_SERVER_LOGS=1 to the complement CI workflow so when tests gets ran, you won't have to wait for a flake or an error to actually see the log output.

And will be happy to do so. The log spam is outrageous, but was needed for log proof since flakes are random and I didn't want to rely on one just get to the actual server logs

Reply:
Sorry, I should have read your description more carefully.

Reply:
I tried to be as verbose as I could manage without beating my head against a wall. I spent hours on that 😄